src/components/application/Ezcounter.vue

```diff
@@ -1,31 +1,60 @@
 <template>
-  <v-container class="my-10">
+  <v-container >
     <v-row justify="center" align="center">
-      <v-col
-        cols="12"
-        md="8"
-        order="2"
-        order-md="1"
-      >
-        <p class="ma-2 text text-center text-justify">
-          {{ $t("ezcounter") }}
-        </p>
+      <v-col cols="12" md="6">
+        <v-row>
+          <v-col cols="12" :align="alignCentered">
```

Comment:
You could use [`align-*` class](https://v2.vuetifyjs.com/en/styles/flex/#flex-align) instead of a switch/case

```suggestion
          <v-col cols="12" class="align-center align-md-stretch">
```